app/services/DisplayPropertiesService.scala

```diff
@@ -132,5 +135,6 @@ case class DisplayProperty(
     propertyName: String,
     propertyType: String,
     unitType: String,
-    summary: String
+    summary: String,
+    futureDateAllowed: Boolean = true
```

Comment:
This will get overridden as we are setting it to false if the attribute is not present (line number 27).
Also, the attribute name `futureDateAllowed` is fine to me but can you check with Phil or Emma before adding it to the DB?

Reply:
I have spoken with Phil. It turns out all the date fields at the moment cannot be a future date, therefore `futureDateAllowed` is no longer needed.